Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        private const char DollarSign = '£';
+
+        /// <summary>
+        /// Cleans up dirtyFileName 
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public static void Clean(string dirtyFile, string cleanedFile)
```

Comment:
Please check for null. That's why the tests are failing.

Reply:
I am not sure what to check for.  Which tests are failing?   The reason that I am asking is that I ran the tests and they all passed as seen in the screenshot below.  Am I missing your of your test code?

![image](https://user-images.githubusercontent.com/59816879/76059054-bb15e800-5f32-11ea-9323-2489917069d2.png)

All the tests seen in the image above correspond to the ones listed in the image below:

![image](https://user-images.githubusercontent.com/59816879/76059092-d7b22000-5f32-11ea-8948-096f9dc1d1b2.png)

Reply:
https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/203/checks?check_run_id=492106013

Those tests are still failing. I guess either the tests here are outdated or your version is. Try to compare and update. :)

Reply:
I have spent almost 1 hour trying to figure out how to compare.  How do I compare?  Are you saying compare Lesson1Tests.cs file in Chapter2/Homework/1 with the one my local working area on my computer for Chapter2/Homework/1a?  (Note: I use to have Chapter2/Homework/1 on my local working area but I could not upload it so Kai said to create a new branch and so I now have Chapter2/Homework1a for this Chapter 2 Homework 1 assignment)

Reply:
Thanks.  With that reply, after a couple of hours, I finally figured out the problem and why I was getting different result.  It was because I had unknowingly stopped checks from being done even though I was clicking Test Explorer to run all the tests.  I also discovered more about how to read this GitHub results in that if I see this "csinn.CSharp-From-Zero-To-Hero Failing after 2m", I know there is a problem to click on Detail which I thought it was part of the Codacy problem about having to ignore Property.  I know, it may not make sense from an expert view such as yourself about what I wrote, but it's how a newbie thinks when sees the new stuff.... at least in my mind.